<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VValvePage</class>
 <widget class="QWidget" name="VValvePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1763</width>
    <height>700</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1501</width>
     <height>681</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>981</width>
     <height>0</height>
    </size>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>325</number>
   </attribute>
   <column>
    <property name="text">
     <string>index</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Description</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Value(ms)</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QToolButton" name="btnOnSR">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1660</x>
     <y>80</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none</string>
   </property>
   <property name="text">
    <string>ON</string>
   </property>
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/Images/Images/SROff.png</normaloff>
     <normalon>:/Images/Images/SROn.png</normalon>
     <disabledon>:/Images/Images/SROn.png</disabledon>:/Images/Images/SROff.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>128</width>
     <height>128</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="btnOffSR">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1660</x>
     <y>170</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none</string>
   </property>
   <property name="text">
    <string>OFF</string>
   </property>
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/Images/Images/SROff.png</normaloff>
     <normalon>:/Images/Images/SROn.png</normalon>
     <disabledon>:/Images/Images/SROn.png</disabledon>:/Images/Images/SROff.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>128</width>
     <height>128</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="btnOnOutput">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1520</x>
     <y>80</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none</string>
   </property>
   <property name="text">
    <string>ON</string>
   </property>
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/Images/Images/Button2.png</normaloff>
     <normalon>:/Images/Images/Button1.png</normalon>
     <disabledon>:/Images/Images/Button1.png</disabledon>:/Images/Images/Button2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>256</width>
     <height>256</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="arrowType">
    <enum>Qt::NoArrow</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="btnOffOutput">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1520</x>
     <y>170</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none</string>
   </property>
   <property name="text">
    <string>OFF</string>
   </property>
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/Images/Images/Button2.png</normaloff>
     <normalon>:/Images/Images/Button1.png</normalon>
     <disabledon>:/Images/Images/Button1.png</disabledon>:/Images/Images/Button2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>256</width>
     <height>256</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="btnStop">
   <property name="geometry">
    <rect>
     <x>1630</x>
     <y>570</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/Images/Images/stop2.png</normaloff>
     <normalon>:/Images/Images/stop2.png</normalon>:/Images/Images/stop2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>256</width>
     <height>256</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="btnSaveFile">
   <property name="geometry">
    <rect>
     <x>1520</x>
     <y>590</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/Images/Images/SaveFile.png</normaloff>:/Images/Images/SaveFile.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>256</width>
     <height>256</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="btnRepeat">
   <property name="geometry">
    <rect>
     <x>1580</x>
     <y>400</y>
     <width>121</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none</string>
   </property>
   <property name="text">
    <string>Repeat</string>
   </property>
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/Images/Images/SwitchOff.png</normaloff>
     <normalon>:/Images/Images/SwitchOn.png</normalon>:/Images/Images/SwitchOff.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>128</width>
     <height>128</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpen">
   <property name="geometry">
    <rect>
     <x>1530</x>
     <y>300</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>1650</x>
     <y>300</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblInfo">
   <property name="geometry">
    <rect>
     <x>1560</x>
     <y>20</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>V----</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
